src/routes/coupon.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: elvis
+ * Date: 2018/11/11
+ * Time: 18:12
+ */
+
+use \Psr\Http\Message\ServerRequestInterface as Request;
+use \Psr\Http\Message\ResponseInterface as Response;
+
+// Get Coupons By User
+$app->get('/api/coupons/{username}', function (Request $request, Response $response, array $args) {
```

Comment:
change this to `/api/coupons/customer/{username}`

Reply:
This reason for this is `'/api/coupons/{username}'` this looks like get coupon by id = username